third_party/2and3/requests/packages/urllib3/__init__.pyi

```diff
@@ -1,8 +1,8 @@
 from typing import Any
 from . import connectionpool
-from . import filepost
-from . import poolmanager
-from . import response
+from . import filepost as filepost
```

Comment:
Is this really needed? That feels like a bug in mypy or the PEP; child modules should always be re-exported.